hack/demo.sh

```diff
@@ -17,24 +17,62 @@
 
 set -euo pipefail
 
+echo "* Installing demo CRDs (ReleasePlan, ReleasePlanAdmission)..."
+kubectl apply -f hack/demo-crds.yaml
+
+echo "* Waiting for CRDs to be ready..."
+kubectl wait --for condition=established --timeout=60s crd/releaseplans.appstudio.redhat.com
+kubectl wait --for condition=established --timeout=60s crd/releaseplanadmissions.appstudio.redhat.com
+kubectl wait --for condition=established --timeout=60s crd/enterprisecontractpolicies.appstudio.redhat.com
+
+echo "* Setting up demo resources (ReleasePlan, ReleasePlanAdmission)..."
+kubectl apply -f hack/demo-resources.yaml
+
+echo "* Waiting for resources to be ready..."
+sleep 2
+
 echo "* Deleting old snapshot..."
 kubectl delete --ignore-not-found -f test-snapshot.yaml
 
 echo "* Creating new snapshot..."
 kubectl create -f test-snapshot.yaml
 
 echo "* Waiting for pod"
-until kubectl get pod -l serving.knative.dev/service=conforma-knative-service -n default &>/dev/null; do sleep 1; done
-kubectl wait --for=condition=Ready pod -l serving.knative.dev/service=conforma-knative-service -n default --timeout=30s
+until kubectl get pod -l app=conforma-knative-service -n default &>/dev/null; do sleep 1; done
+kubectl wait --for=condition=Ready pod -l app=conforma-knative-service -n default --timeout=30s
```

Comment:
Maybe this is easily replaced by hack/wait-for-ready.

Reply:
ack.